tests/metadata_test.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for metadata.py"""
+
+import string
+import random
+import tempfile
+import numpy as np
+import pytest
+
+from git_theta import metadata
+from tests.testing_utils import (
+    random_lfs_metadata,
+    random_tensor_metadata,
+    random_theta_metadata,
+    random_param_metadata,
+    random_metadata,
+)
+
+
+def metadata_equal(m1, m2):
```

Comment:
What is the output we get when these aren't equal? Pytest normally shows info about what doesn't match, is that available with this? If not can we add an issue to eventually have better integration with asserts for seeing what the mismatch is?